alerts/proxy_drop_ip.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import QueryStringMatch, SearchQuery, TermMatch
+import re
+
+
+class AlertProxyDropIP(AlertTask):
+    def main(self):
+        search_query = SearchQuery(minutes=20)
+
+        search_query.add_must([
+            TermMatch('category', 'squid'),
+            TermMatch('tags', 'squid'),
+            TermMatch('details.proxyaction', 'TCP_DENIED/-')
+        ])
+
+        # Match on 1.1.1.1, http://1.1.1.1, or https://1.1.1.1
+        # This will over-match on short 3-char domains like foo.bar.baz.com, but will get weeded out below
+        ip_regex = '/.*\..{1,3}\..{1,3}\..{1,3}(:.*|\/.*)/'
```

Comment:
I think this needs to be a bit more strict, I believe this false positives on the following details.destionation:
```
http://mirrors.cat.pdx.edu/centos/7.3.1611/os/x86_64/repodata/repomd.xml
```

Reply:
Did you note the comment on line 25?  I pretty much say this much and validate later to prevent false positives.

Reply:
Ahhh, no I did not.